File: problemas/matematica/1585/1585.c
#include <stdio.h>

int main()
{
    int N, x, y;

    scanf("%d", &N);

    for (int i = 0; i < N; ++i)
    {
        scanf("%d %d", &x, &y);

        printf("%d cm2\n", (x * y) / 2);
    }

    return 0;
}

File: problemas/matematica/1585/1585.go
package main

import (
    "fmt"
)

func main() {
    var n, x, y int

    fmt.Scanf("%d", &n)
    for i := 0; i < n; i++ {
        fmt.Scanf("%d %d", &x, &y)
        fmt.Printf("%d cm2\n", (x*y)/2)
    }
}
